CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
```

Comment:
This has references to python and such in it, would be good to update to reference .NET

Could also be good to add a github PR template like we have in some of our other SDKs.